### ISSUE TITLE
String concatenation causes an error

### ISSUE THREAD

**Issue by GrizliK1988** [akameco/babel-plugin-react-intl-auto#46]
* version:
* `node` version: v9.4.0
* `npm` (or `yarn`) version: npm 5.6.0 yarn 1.6.0

**Do you want to request a _feature_ or report a _bug_?:**
_bug_

**What is the current behavior?:**
Following code
```
import { defineMessages } from 'react-intl'

export default defineMessages({
  successMessage: 'Some long string  string  string  string  string  string  string  string' +
    '  string  string '
})
```

Produces an error:
```
[React Intl] An `id` must be provided to format a message.
```

**What is the expected behavior?:**
String concatenation should be handled without an error

**Suggested solution:**
Can't propose a solution for now.

**Comment by akameco** [comment]
Consider using `Template literals`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

**Comment by akameco** [comment]
Fixed #47
